### DISCUSSION TITLE
Update the Plugin in Unity Assets Store

### DISCUSSION THREAD

**Discussion by SirinC** [mlavik1/UnityVolumeRendering#223] [Q&A] [answered]
Hi,

I would like to ask, if you can update the Plugin in the Unity Assets Store? I guess there is currently an old version of your Volume Renderer.

**Comment by mlavik1** [comment, marked as answer]
Thanks for the reminder! I'll see if I get time to do it next week :) This week was kind of crazy :sweat_smile: 
Although, there haven't been that many big updates since last release. Mostly some minor bugfixes, plus this: #208 
I'm hoping to find some more time soon to work on some other features.

**Comment by mlavik1** [comment]
New version has been published! Managed to get two PRs in first, with support for saving VolumeRenderedObject as prefab, 3D TIFF file support and fixed SimpleITK image sequence import :)
